docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: '3'
```

Comment:
I just had to learn it for development in Laravel and I bought a Udemy course and I was amazed at how awesome it is. We will deploy so many instances of that backend it will scale a little bit too well!